@here/harp-mapview/lib/WorkerBasedDecoder.ts

```diff
@@ -76,15 +79,19 @@ export class WorkerBasedDecoder implements ITileDecoder {
     async connect(): Promise<void> {
         await this.workerSet.connect(WorkerServiceProtocol.WORKER_SERVICE_MANAGER_SERVICE_ID);
         if (!this.m_serviceCreated) {
-            await this.workerSet.broadcastRequest(
-                WorkerServiceProtocol.WORKER_SERVICE_MANAGER_SERVICE_ID,
-                {
-                    type: WorkerServiceProtocol.Requests.CreateService,
-                    targetServiceType: this.decoderServiceType,
-                    targetServiceId: this.serviceId
-                }
-            );
-            this.m_serviceCreated = true;
+            try {
+                await this.workerSet.broadcastRequest(
+                    WorkerServiceProtocol.WORKER_SERVICE_MANAGER_SERVICE_ID,
+                    {
+                        type: WorkerServiceProtocol.Requests.CreateService,
+                        targetServiceType: this.decoderServiceType,
+                        targetServiceId: this.serviceId
+                    }
+                );
+                this.m_serviceCreated = true;
+            } catch (e) {
+                logger.error(e);
```

Comment:
What if `connect` fails ?
Please surround whole fun with `try/catch`.

Some meaningful message like 
```
logger.error("Unable to connect to tiler/decoder service", e)
```
would be warmly welcomed too.

(same applies to second instance of this change)

Reply:
NVM, addressed in #761